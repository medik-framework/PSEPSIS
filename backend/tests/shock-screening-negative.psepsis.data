{
  "Normal Saline": 45,
  "MAP": 60,
  "Epinephrine": 3,
  "Base Excess": -3,
  "Arterial Lactate": 340
}
