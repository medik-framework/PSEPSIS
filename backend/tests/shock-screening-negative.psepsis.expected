[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Normal Saline"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "MAP"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "BP Sys"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Epinephrine"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Norepinephrine"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Dopamine"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Base Excess"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Arterial Lactate"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Urine Output"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Capillary Refill"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Temp"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Core Temp"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ShockDiagnosis",
    "args": [false]
  }
]
